src/oracle.js

```diff
@@ -218,6 +218,7 @@ const updateOracle = ({
   ipfsGateway = DEFAULT_IPFS_GATEWAY,
   oracleApp,
   oracleContract,
+  targetBlockchains,
```

Comment:
we should validate `targetBlockchain` as an optional array of chainId before using it (an incorrect targetBlockchain will cause the geaneric-oracle-dapp to fail)
I may propose an implementation depending on my remaining time